python/tvm/relay/op/strategy/arm_cpu.py

```diff
@@ -235,6 +235,28 @@ def conv2d_strategy_arm_cpu(attrs, inputs, out_type, target):
                     wrap_topi_schedule(topi.arm_cpu.schedule_depthwise_conv2d_nhwc),
                     name="depthwise_conv2d_nhwc.arm_cpu",
                 )
+
+            # Optimized special case depthwiseConv2D operation. Requires a 3x3 kernel, a
+            # NHWC layout, a HWOI kernel layout (which we would ideally rearrange), no dilation,
+            # "SAME" padding, int8 inputs and outputs, the same number of input and output
+            # channels, and for that channel count to be divisible by 4.
+            #
+            # Additional work could remove some of these restrictions.
+
+            elif (
+                target.features.has_dsp
```

Comment:
Does it make sense to introduce out_dtype check as well up here?

Reply:
Added a check and adjusted the above comment.